firmware/user/modes/mode_rssi.c

```diff
@@ -426,17 +646,47 @@ static void ICACHE_FLASH_ATTR rssiUpdate(void* arg __attribute__((unused)))
                         }
 
                         // Start NTP if we have an IP address and it hasn't been started yet
-                        if(ipi.ip.addr != 0 && false == rssi->sntpInit)
+                        if(ipi.ip.addr != 0 && false == rssi->netInit)
                         {
                             RSSI_PRINTF("Init SNTP\n");
-                            rssi->sntpInit = true;
+                            rssi->netInit = true;
                             sntp_setservername(0, "time.google.com");
                             sntp_setservername(1, "time.cloudflare.com");
                             sntp_setservername(2, "time-a-g.nist.gov");
                             sntp_init();
                             sntp_set_timezone(rssi->tz); // Eastern
+
+                            //Set up other sockets.
+                            memset( &rssi->server_udp_leds, 0, sizeof( rssi->server_udp_leds ) );
```

Comment:
If you switch to the pixelflut submode too fast, this won't get initialized. It should probably be moved before the switch statement.